snapcraft.yaml

```
@@ -8,7 +8,8 @@ license: GPL-2.0
 
 version: '0.1'
 confinement: strict
-base: core18
+type: base
```

We should instead have this:

```suggestion
base: bare
```